### DISCUSSION TITLE
New Source: Reddit

### DISCUSSION THREAD

**Discussion by sherifnada** [airbytehq/airbyte#33181] [New Connector Request]
## Tell us about the new integration you’d like to have
I would love to be able to use Airbyte to pull data from Reddit for sentiment analysis applications. 

https://www.reddit.com/dev/api/

## Describe the context around this new integration
Various applications need to scrape social media to perform studies, aggregation, or sentiment analysis. Usually, this must be done by writing a custom client. But I would love for it to "just work" 

## Describe the alternative you are considering or using
Writing my own

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1200367912513076/1200368155103319) by [Unito](https://www.unito.io)

**Comment by btkcodedev** [comment]
Analysis:
Outh reference
https://github.com/reddit-archive/reddit/wiki/OAuth2-Quick-Start-Example

API reference:
https://www.reddit.com/dev/api/oauth

**Comment by btkcodedev** [comment]
Useful streams:
/me
/search?q=
/subreddits/search?q=
/message/inbox

**Comment by btkcodedev** [comment]
Started working on https://github.com/airbytehq/airbyte/issues/44561

[a comment by an automated account is not shown]
